### ISSUE TITLE
Login loops when using https in config

### ISSUE THREAD

**Issue by goldug** [SergiX44/XBackBone#488]
**System Info**
+ PHP Version: 7.4
+ XBackBone Version: 3.6.0
+ Webserver: [Apache/Nginx/...] Apache
+ Database backend: [SQLite/Mysql/...] MySQL

**Describe the bug**
When I have config.php set to https in the url I cannot login. It just redirects me to the login page.

**To Reproduce**
1. Change the config.php to use https instead of http
2. Try to login

**Expected behavior**
I expect to be logged in.

**Logs**
The only thing in the log is this: [2022-08-26 14:55:36] app.INFO: User admin logged in. [] []

**Comment by SrS2225a** [comment]
Sounds like it is something with your web server. Do you have your web server address using https as well?

**Comment by goldug** [comment]
Yeah, I'm using Virtualmin to set up the server, according to that it should work.
I'm using a subdomain for XbackBone as well as a couple of other sites on my server and XbackBone is the only one not working with ssl atm.

**Comment by SrS2225a** [comment]
I'm not all that familiar with virtualmin. I know more about nginx and apache, so I can't be as much help there. But do you happen to have all the required php modules xbackbone needs installed and enabled? Please make sure you do. If that's not the issue,  I would like if you could send xbackbone latest log with the error. It may be saying something there for us

**Comment by stefbrouwers** [comment]
had the same issue fixed it with (#347)
